llmfoundry/registry.py

```diff
@@ -389,6 +390,26 @@
     description=_save_planners_description,
 )
 
+_tp_strategy_description = (
```

Comment:
the other registries are plural, so lets make this `tp_strategies`?

Reply:
done!